dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/association/jdbc/JdbcOrgUnitAssociationsStore.java

```diff
@@ -119,7 +122,14 @@ public SetValuedMap<String, String> getOrganisationUnitsAssociations( Set<String
 
     private Set<String> getUserOrgUnitPaths()
     {
-        Set<String> allUserOrgUnitPaths = currentUserService.getCurrentUserOrganisationUnits().stream()
+        Set<OrganisationUnit> currentUserOrganisationUnits = currentUserService.getCurrentUserOrganisationUnits();
```

Comment:
Just curious, why is this necessary? Are we getting detached or cached org units from the current user service?

Reply:
Yes, right now the user is detached after re-trivial on login. This is possible the disable very easy, right now I think we should consider this an ongoing experiment.